tools/WebhookProcessor/github_webhook_processor.php

```diff
@@ -561,7 +561,7 @@ function has_tree_been_edited($payload, $tree){
 	get_diff($payload);
 	//find things in the _maps/map_files tree
 	//e.g. diff --git a/_maps/map_files/Cerestation/cerestation.dmm b/_maps/map_files/Cerestation/cerestation.dmm
-	return $github_diff !== FALSE && preg_match('/^diff --git a\/' . preg_quote($tree, '/') . '/m') !== FALSE;
+	return $github_diff !== FALSE && preg_match('/^diff --git a\/' . preg_quote($tree, '/') . '/m', $github_diff) !== FALSE;
```

Comment:
This is completely my fault, should be ```!== 0``` http://php.net/manual/en/function.preg-match.php#refsect1-function.preg-match-returnvalues